crates/holochain_p2p/tests/tests/blocks.rs

```diff
@@ -3,37 +3,35 @@ use holo_hash::{
     fixt::{AgentPubKeyFixturator, DhtOpHashFixturator, DnaHashFixturator},
     DnaHash,
 };
-use holochain_keystore::test_keystore;
+use holochain_keystore::{test_keystore, MetaLairClient};
 use holochain_p2p::{
     actor::DynHcP2p, event::MockHcP2pHandler, spawn_holochain_p2p, HolochainP2pConfig,
+    HolochainP2pLocalAgent,
 };
 use holochain_state::{block::get_all_cell_blocks, prelude::test_conductor_db};
+use holochain_timestamp::{InclusiveTimestampInterval, Timestamp};
 use holochain_types::{
     db::{DbKindConductor, DbKindDht, DbKindPeerMetaStore, DbWrite},
-    prelude::{CellBlockReason, CellId},
+    prelude::{Block, CellBlockReason, CellId},
 };
 use holochain_zome_types::block::BlockTarget;
-use kitsune2_api::DynBlocks;
+use kitsune2_api::{AgentInfo, AgentInfoSigned, DhtArc, DynBlocks};
 use std::sync::Arc;
 
 #[tokio::test(flavor = "multi_thread")]
 async fn cell_blocks_are_committed_to_database() {
-    let conductor_db = test_conductor_db().to_db();
-    let conductor_db2 = conductor_db.clone();
-    let config = HolochainP2pConfig {
-        get_conductor_db: Arc::new(move || {
-            let conductor_db2 = conductor_db2.clone();
-            Box::pin(async move { conductor_db2 })
-        }),
-        k2_test_builder: true,
-        ..Default::default()
-    };
-    let actor = spawn_holochain_p2p(config, test_keystore()).await.unwrap();
-    let cell_id = CellId::new(fixt!(DnaHash), fixt!(AgentPubKey));
+    let conductor_db = DbWrite::test_in_mem(DbKindConductor).unwrap();
```

Comment:
Why not `test_conductor_db`?

Reply:
No big reason, just that this runs in memory and doesn't create a file.